Dockerfile

```diff
@@ -51,6 +48,23 @@ RUN cd /serialbox && \
     make install && \
     /bin/rm -rf /serialbox
 
+# Install Google Cloud
+RUN apt-get update && apt-get install -y  apt-transport-https ca-certificates gnupg curl gettext
+
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list &&\
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+
+
+RUN apt-get update && apt-get install -y google-cloud-sdk jq python3-dev python3-pip kubectl gfortran
```

Comment:
Would it make sense to group the non google cloud stuff in the apt-get install at the very beginning of the Dockerfile? It seems that python3-dev, python3-pip, and gfortran are redundant with the specification above.